src/simplecoin_test.sol

```diff
@@ -56,8 +55,8 @@ contract SimplecoinTest is Test {
         // TODO: check authority setup
     }
 
-    function testCreatedSimplecoinHasNoAuthority() {
-        assertEq(0, coin.authority());
+    function testCreatorHasAuthority() {
+        assertEq(DSAuth(coin.authority()).owner(), this);
```

Comment:
this accidentally works, but is an artifact from when these were separate contracts that were actually chained like this

could be clearer with 2 separate asserts (owner and authority are both this)